src/components/organisms/WorldEntranceForm/WorldEntranceForm.tsx

```diff
@@ -149,7 +183,8 @@ export const WorldEntranceForm: React.FC<WorldEntranceFormProps> = ({
     isSubmitting ||
     isDirtyCode ||
     isDirtyProfile ||
-    isDirtyEntrance
+    isDirtyEntrance ||
+    isDirtyButtons
```

Comment:
This is out of place. Since buttons are part of the entrance, adding or removing or changing a button should make `idDirtyEntrance` be true